common/lib/capa/capa/capa_problem.py

```diff
@@ -519,15 +520,27 @@ def do_targeted_feedback(self, tree):
                 if choice.get('name') == student_answer:
                     expl_id_for_student_answer = choice.get('explanation-id')
                 if choice.get('correct') == 'true':
+                    choice_correctness = _('Correct')
```

Comment:
@alisan617 This isn't working because choice_correctness gets overwritten every time through the loop. You will need to define choice_correctness as _('Incorrect') outside of the loop (like solution_id is), and then only set it to _('Correct') if choice.get('correct') == 'true' (again, like solution_id).

You can test this out by making a targeted feedback item that maps to a correct item, such as:

````
<problem>
<p>What Apple device competed with the portable CD player?</p>
<multiplechoiceresponse targeted-feedback="">
 <choicegroup type="MultipleChoice">
   <choice correct="false" explanation-id="feedback1">The iPad</choice>
   <choice correct="false" explanation-id="feedback2">Napster</choice>
   <choice correct="true" explanation-id="correct">The iPod</choice>
   <choice correct="false" explanation-id="feedback3">The vegetable peeler</choice>
 </choicegroup>
</multiplechoiceresponse>
  
  <targetedfeedbackset>
  <targetedfeedback explanation-id="feedback1">
    <div class="detailed-targeted-feedback">
      <p>Targeted Feedback</p>
      <p>The iPad came out later and did not directly compete with portable
         CD players.</p>
    </div>
  </targetedfeedback>
  <targetedfeedback explanation-id="feedback2">
    <div class="detailed-targeted-feedback">
      <p>Targeted Feedback</p>
      <p>Napster was not an Apple product.</p>
    </div>
  </targetedfeedback>
  <targetedfeedback explanation-id="feedback3">
    <div class="detailed-targeted-feedback">
      <p>Targeted Feedback</p>
      <p>Vegetable peelers do not play music.</p>
    </div>
  </targetedfeedback>
    <targetedfeedback explanation-id="correct">
    <div class="detailed-targeted-feedback">
      <p>Targeted Feedback</p>
      <p>You are correct!</p>
    </div>
  </targetedfeedback>
 </targetedfeedbackset>

 <solution explanation-id="correct">
  <div class="detailed-solution">
   <p>The iPod directly competed with portable CD players.</p>
  </div>
 </solution>
 </problem>